### ISSUE TITLE
New test for volumes

### ISSUE THREAD

**Issue by devblackops** [Ticketmaster/poshspec#14]
I'm thinking a new test for volumes would be useful. If people agree I'll create a PR implementing it.

``` powershell
describe 'Storage' {
    context 'Capacity' {
        volume 'c' sizeremaining { should begreaterthan 1073741824 }
    }
    context 'Health' {
        volume 'c' operationstatus { should be 'ok' }
        volume 'c' healthstatus { should be 'healthly' }
    }
}
```

**Comment by cdhunt** [comment]
I think it would be a good addition.

**Comment by beaudryj** [comment]
I got a head start on this

**Comment by devblackops** [comment]
What needs to happen for PR #15 to be completed that implements this?

**Comment by cdhunt** [comment]
I need @beaudryj to resubmit it with just the one change. Right now the PR contains a bunch of old changes that are causing a conflict.

**Comment by beaudryj** [comment]
I have been slacking real hard, I'll try to get that in. Sorry Brandon

**Comment by devblackops** [comment]
[cracks whip] @beaudryj  :smile:

**Comment by michaeltlombardi** [comment]
Should this issue be closed by #30?

**Comment by cdhunt** [comment]
Yup. Thanks, @devblackops and @michaeltlombardi .
